app/static/js/vendor/jquery/jquery.codezero.js

```diff
@@ -150,13 +150,11 @@ var cachedFuzzyMatch;
  * Cache the results of a RegExp match.
  * @type {Function}
  */
-if ('undefined' === !typeof _) {
-    cachedFuzzyMatch = _.memoize(function (pattern) {
-        return new RegExp(pattern.split("").reduce(function (a, b) {
-            return a + '[^' + b + ']*' + b;
-        }), 'i');
-    });
-}
+cachedFuzzyMatch = _.memoize(function (pattern) {
```

Comment:
Huh .. that's weird ... I was the one who had implemented this long ago ... Could you check why ?